plugins/input/http.md

```diff
@@ -327,6 +348,45 @@ curl -X POST -F 'json={"message":"foo+bar"}' http://localhost:9880/app.log
 
 -   [Input Plugin Overview](/plugins/input/README.md)
 
+## Tips
+
+
+### How to Enable TLS Encryption
+
+Since v1.5.0, `in_http` support TLS tranport. Here is configuration example with HTTPS client.
+
+- in_tcp
+
+```
+<source>
+  @type http
+  bind 0.0.0.0
+  <transport tls>
+    insecure true
```

Comment:
insecure example is not good.

Reply:
fixed not to use insecure directive.
https://github.com/fluent/fluentd-docs-gitbook/pull/50/commits/96f993207361a4297bc103c9094aa1507c29f6e6